### ISSUE TITLE
TypeError in req.ip.hostname

### ISSUE THREAD

**Issue by tcelestino** [raaymax/planigale#18]
Hi there.

I'm using [tool] to study how to create a web server with Deno in a practical way, and I've encountered a TypeScript error. I'm using the middleware construction example from the README.md to capture log information, and when I use `req.ip.hostname`, I get the error: "Property 'hostname' does not exist on type 'string | NetAddr'."

My suggestion would be to change this:
```typescript
export class Req {
  /** IP address of the client that sent the request. */
  ip: Deno.NetAddr;
  /** Route object to which the request is directed. */
  route?: Route;
  ...
```
In this case, it wouldn't be necessary to do `req.ip as Deno.NetAddr;` to use the `hostname`. Would removing the `string` type have significant implications?"

**Comment by raaymax** [comment]
Sorry, for late answer I was out for some time.
You are absolutely right, I just fixed your issue in version 0.6.6.

**Comment by tcelestino** [comment]
@raaymax, thanks!!
